setup.cfg

```diff
@@ -1,25 +0,0 @@
-[flake8]
-max-line-length = 88
-ignore = E203, E402, E731, E704, W503, W504, W291, W293
```

Comment:
I don't fully know how this works, but shouldn't we be ignoring all of these in the `pyproject.toml`? In the `pyproject.toml` it seems to only be `E203`, `E402` and `E731`